### PR TITLE
Add a deployment adapter for Fly.io

### PR DESCRIPTION
**Pull request by jsierles**
This adapter implements the minimum work to check that a project is setup for deployment on Fly.io. From there, developers will be directed to Fly Redwood deployment docs.

Setting up CI against Fly can be done in a separate PR.

### CODE REVIEW

File: docs/docs/deploy/fly.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Deploy to Fly.io
+
+
+Fly.io allows you to run your full stack apps (and databases!) all over the world. No ops required. A simple Redwood app can run permanently (no time limits) on the Fly free tier with a 1GB Postgresql database, or a 1GB dedicated storage volume for SQLite.
+
+## FLy.io tl;dr Deploy
```
Comment (pavelloz) [inline docs/docs/deploy/fly.md:6]: `FLy.io` -> `Fly.io` :)

File: docs/docs/deploy/fly.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Deploy to Fly.io
+
+
+Fly.io allows you to run your full stack apps (and databases!) all over the world. No ops required. A simple Redwood app can run permanently (no time limits) on the Fly free tier with a 1GB Postgresql database, or a 1GB dedicated storage volume for SQLite.
```
Comment (pavelloz) [inline docs/docs/deploy/fly.md:4]: `Redwood` -> `RedwoodJS`

File: docs/docs/deploy/fly.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+## FLy.io tl;dr Deploy
+
+If you simply want to experience the Fly.io deployment process, including a Postgres or SQLite database, you can do the following:
```
Comment (pavelloz) [inline docs/docs/deploy/fly.md:8]: Before and above you used Postgresql as name for PG, so i assume "Postgres" is a typo.

**Comment by CarlQLange** [comment]
It would be nice to see this merged :)

**Comment by Josh-Walker-GM** [comment]
I'm going to close this PR. **This is not because we don't want this!** It is just that this PR is now very old and we'd likely benefit from starting fresh if we did go forward with implementing a wrapper around the fly.io CLI.

**Comment by Josh-Walker-GM** [comment]
Okay perhaps I was a little too hasty with the close button. I should have asked if @jsierles this is still something you'd want to get in? If so I can then look to find time for it.

**Pull request closed without merging**